projects/tuist/features/step_definitions/init.rb

```diff
@@ -8,3 +8,7 @@
 When(/^I initialize a (.+) application named ([a-zA-Z\-_]+)$/) do |platform, name|
   system(@tuist, "init", "--path", @dir, "--platform", platform, "--name", name)
 end
+
+When(/^I initialize a project from the template (.+)$/) do |template_url|
```

Comment:
```suggestion
When(/^I initialize a project with (.+) template$/) do |template_url|
```
I'd just stay more consistent with [this](https://github.com/tuist/tuist/pull/3796/files#diff-4334ed6a9047766121e1bda506bfb3931c61abe4596415feb4667ba98dce0344R3) method, otherwise, looks good 👍